pytorch_toolkit/instance_segmentation/configs/instance-segmentation-0002.py

```diff
@@ -0,0 +1,33 @@
+_base_ = '../../../external/mmdetection/configs/mask_rcnn/mask_rcnn_r50_caffe_fpn_mstrain-poly_3x_coco.py'
```

Comment:
Is it Ok to use such relative paths?

Reply:
Shouldn't these models be compliant with Model Templates? Do they?

Reply:
These models have to be complaint with Model Templates (https://github.com/openvinotoolkit/training_extensions/pull/335/).

During runtime of a platform model folders are usually copied outside of OTE repo folder, so that such relative paths are forbidden.

Reply:
I thought that we should use submodule placed in `training_extensions/external/mmdetection`. If it is true, then relative paths of base configuration files are correct, I have checked training and validation commands. Should it be full config-files without `_base_` option? In config may be incorrect relative path of `data_root` directory which depends on a running directory.

Reply:
There are two cases at least we have to support both.
1. Training from repo. In that case what you have now is OK.
2. Training in a platform where model folder (config.py and other stuff) can be moved to some place and mmdetection can be  pre-built in some docker image. Here we can't use `_base_`. `data_root` in this case will be definitely changed. So please copy-paste content of `_base_` config right here.

Reply:
I guess you can keep base config in a separate file, but have it here. @Ilya-Krylov am I right?

Reply:
fixed with complete configuration files.